hazelcast/src/main/java/com/hazelcast/jet/pipeline/JdbcSinkBuilder.java

```diff
@@ -243,7 +244,7 @@ public Sink<T> build() {
         }
         if (jdbcUrl != null) {
             String connectionUrl = jdbcUrl;
-            dataSourceSupplier = () -> new DataSourceFromJdbcUrl(connectionUrl);
+            dataSourceSupplier = () -> DataStoreSupplier.closing(new DataSourceFromJdbcUrl(connectionUrl));
```

Comment:
Can this wrapping be unneeded, since `DataSourceFromJdbcUrl` is not `Closable` ?

Reply:
Fixed: https://github.com/hazelcast/hazelcast/pull/22372/commits/53c89bb17b45d8f9ddeef702d5404a9eed9ebb72